tests/trio_options.py

```diff
@@ -1,3 +1,6 @@
+app = None
+
+
 @app.route  # type: ignore
 async def f():
     print("hello world")
```

Comment:
With these changes this file passes linters, so I don't have to exclude it in the rules. It was previously caught by the `tests/trio_*.py` pattern, but since it's not an eval'd file I didn't move it into the eval directory.
Although given the support for `#ARG` it's actually pretty simple to remove the dedicated test for this and make it an eval'd test file - but I'll leave that for a later TODO. Otherwise this file would probably benefit from a comment and/or another directory for `other_test_files` (or something)